core/src/main/scala/sttp/client4/logging/LogContext.scala

```diff
@@ -0,0 +1,53 @@
+package sttp.client4.logging
+
+import sttp.client4.Response
+import sttp.model.{HeaderNames, RequestMetadata}
+
+import scala.collection.mutable
+import scala.concurrent.duration.Duration
+
+trait LogContext {
+  def ofRequest(request: RequestMetadata): Map[String, Any]
```

Comment:
I'm not a native speaker, but shouldn't this be `forRequest` and `forResponse`?

Reply:
I'm not a native either, I think the 2 are correct depending on the meaning you want to give.

I'm changing it to `for` because that suits me too.